src/components/staking/Undelegations.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <TransactionList
+      :transactions="unbondingTransactions"
+      :address="session.address"
+      :validators="yourValidators"
+    />
+  </div>
+</template>
+
+<script>
+import { mapState, mapGetters } from "vuex"
+import TransactionList from "transactions/TransactionList"
+import { messageType } from "transactions/messageTypes"
+
+export default {
+  name: `undelegations`,
+  components: {
+    TransactionList
+  },
+  computed: {
+    ...mapState([`session`]),
+    ...mapGetters([`flatOrderedTransactionList`, `yourValidators`]),
+    unbondingTransactions: ({ flatOrderedTransactionList } = this) =>
+      flatOrderedTransactionList.filter(
+        transaction => transaction.type === messageType.UNDELEGATE
```

Comment:
You need to check the date too. After it's unbonded it's still an undelegate type.